src/components/Dashboard/TodoList.vue

```diff
@@ -1088,17 +1104,16 @@ export default class TodoList extends Mixins(AllowableActionsMixin, DateMixin, E
     const header = filing.header
 
     if (dissolution && business && header) {
-      // don't allow resuming a draft if not in good standing
-      if (this.isStatusDraft(header) && !this.isGoodStanding) {
-        task.enabled = false
-      }
+      // non-staff can only delete item when NIGS
+      const showDeleteOnly = (!this.isRoleStaff && !this.isGoodStanding)
```

Comment:
There are 3 conditions to show the "Delete draft" button only:
1. user must be non-staff (ie, staff get the usual buttons)
2. business must be NIGS (otherwise user gets the usual buttons)
3. this filing must be in draft status (this is checked in the template so that all "if draft" conditions are together and clear)

Ditto for Dissolution and Special Resolution filings below.

Reply:
I changed this to a method called from the template (see another comment below) but same idea.